### PR TITLE
Deployment related fixes 

### PR DESCRIPTION
**Pull request by MiguelPires**

### CODE REVIEW

File: blockchain-scrapers/createStack.sh [hunk 1]
```diff
@@ -24,13 +24,13 @@ function deploy {
 	case $1 in
```
Comment (frranck) [inline blockchain-scrapers/createStack.sh]: Shouldnt we get rid of this script ? it's covered bu push.sh

File: blockchain-scrapers/push.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/mute/secrets/push.sh
```
Comment (frranck) [inline blockchain-scrapers/push.sh]: what's that ?

File: blockchain-scrapers/blockchains/eos/docker-compose.eos.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+services:
+  eos-node:
+    image: ${DOCKER_HUB_LOGIN}/${STACKNAME}_node-eos:latest  
+    build:
+      context: $GOPATH
+      dockerfile: $GOPATH/src/github.com/diadata-org/api-golang/blockchain-scrapers/blockchains/eos/node/Dockerfile-node-eos
+    ports:
+      - 8888:8888
+      - 9876:9876
+    volumes:
```
Comment (frranck) [inline blockchain-scrapers/blockchains/eos/docker-compose.eos.yml]: do we want that exposed ? cause we have the passwords public...

File: blockchain-scrapers/blockchains/neo/docker-compose.neo.yml [hunk 4]
```diff
@@ -7,8 +7,7 @@ services:
     ports:
       - "10332:10332"
     volumes:
```
Comment (frranck) [inline blockchain-scrapers/blockchains/neo/docker-compose.neo.yml]: same we prob dont want to expose the port to the outside world.

Comment (MiguelPires) [reply blockchain-scrapers/blockchains/neo/docker-compose.neo.yml]: Fixed in the latest commit

**Pull request merged**